public/app/features/alerting/unified/components/rules/CloudRules.tsx

```diff
@@ -75,7 +75,7 @@ export const CloudRules = ({ namespaces, expandAll }: Props) => {
               <div />
             )}
             <Stack gap={1}>
-              {canMigrateToGMA && <MigrateToGMAButton />}
+              {canMigrateToGMA && hasSomeResults && <MigrateToGMAButton />}
```

Comment:
Also incidental change - in the old list view, when there are no data source rules, we shouldn't show the import button